soc/nxp/imx/imx8m/a53/soc.c

```diff
@@ -58,7 +58,7 @@ static void soc_rdc_init(void)
 
 #endif
 
-static void soc_clock_init(void)
+void __weak soc_clock_init(void)
```

Comment:
Be consistent and use `__weak`  before `void` always, like `__weak void clock_init()`